Day-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Button.css';
+
+const NumberButton = (props) => {
+    return (        
+        <button className={`number-button ${props.buttonStyle}`}>{props.text}</button>        
```

Comment:
You were passing the prop named `bgColor` but then reading it from here as `props.buttonStyle` this is the reason why your `.num-button` class or its styles were not being applied on this element. How did I know? I used the react dev tools to find out the name of the props that were being passed.